cmd/cli/job/describe.go

```diff
@@ -75,6 +75,10 @@ func NewDescribeCmd() *cobra.Command {
 			return o.run(cmd, args, api)
 		},
 	}
+
+	jobCmd.SilenceUsage = true
+	jobCmd.SilenceErrors = true
```

Comment:
For all cases where a change is made like this, please do so in the `&cobra.Command{` block to keep things clean. So:
```go
jobCmd := &cobra.Command{
		Use:     "describe [id]",
		Short:   "Get the info of a job by id.",
		Long:    describeLong,
		Example: describeExample,
		SilenceUsage: true,
                SilenceErrors: true,
		Args:    cobra.ExactArgs(1),
		RunE: func(cmd *cobra.Command, args []string) error {
			// initialize a new or open an existing repo merging any config file(s) it contains into cfg.
			cfg, err := util.SetupRepoConfig(cmd)
			if err != nil {
				return fmt.Errorf("failed to setup repo: %w", err)
			}
			// create an api client
			api, err := util.GetAPIClientV2(cmd, cfg)
			if err != nil {
				return fmt.Errorf("failed to create api client: %w", err)
			}
			return o.run(cmd, args, api)
		},
	}

Reply:
Aah thanks ! This saves so much !